kong.yaml

```diff
@@ -86,9 +86,50 @@ update:
     use-tag: true
 
 test:
+  environment:
+    contents:
+      packages:
+        - bash
+        - curl
+        - jq
+        - wait-for-it
   pipeline:
-    - runs: |
+    - name: "Basic Test"
+      runs: |
         kong version | grep -i ${{package.version}}
         kong prepare --v 2>/dev/null | grep -i "preparing nginx"
-        kong roar | grep -i "Monolith destroyer"
+    - name: "Test Kong Gateway functionality"
+      uses: test/daemon-check-output
+      with:
+        setup: |
+          mkdir -p /tmp/kong_prefix
+          cat <<'CFG' >/tmp/kong.yml
+          _format_version: "3.0"
+          services:
+            - name: stub
+              url: http://localhost
+              routes:
+                - name: root
+                  paths: [ / ]
+          CFG
+
+          cat <<'SH' >/tmp/run-kong.sh
+          #!/usr/bin/env bash
+          set -euo pipefail
+          export KONG_DATABASE=off
+          export KONG_DECLARATIVE_CONFIG=/tmp/kong.yml
+          export KONG_ADMIN_LISTEN=127.0.0.1:8001
+          export KONG_PROXY_LISTEN=127.0.0.1:8000
+          kong prepare -p /tmp/kong_prefix
+          exec kong start -v -p /tmp/kong_prefix
```

Comment:
We can keep only `kong prepare -p /tmp/kong_prefix` in the `setup:`. And move the `kong start -v -p /tmp/kong_prefix` to `start:` field.